style/ruby/README.md

```diff
@@ -11,6 +11,8 @@ Ruby
 * Avoid explicit return statements.
 * Avoid using semicolons.
 * Avoid bang (!) method names. Prefer descriptive names.
+* Avoid defining classes and modules using shorthand nesting (class A::B)
```

Comment:
I sort-off want to keep `Avoid` in there to emphesize that we couldn't use `A::B` unless it's actually needed